includes/wizards/class-dashboard.php

```diff
@@ -116,6 +116,13 @@ protected function get_dashboard() {
 				'description' => Wizards::get_description( 'popups' ),
 				'status'      => 'enabled',
 			],
+			[
+				'slug'        => 'support',
+				'name'        => Wizards::get_name( 'support' ),
+				'url'         => Wizards::get_url( 'support' ),
+				'description' => Wizards::get_description( 'support' ),
+				'status'      => 'enabled',
+			],
```

Comment:
The inclusion of this link should be conditional, based on whether the Wizard [is configured](https://github.com/Automattic/newspack-plugin/blob/ea2e9a89dd11c94e3ea5f98a4fa19830357144dd/includes/wizards/class-support-wizard.php#L395) (has to have necessary env variables set)

Reply:
Would https://github.com/Automattic/newspack-plugin/pull/650/commits/3cd635cdf9afe309345a6b929a4bd2a3a1e79e18 do the trick here?